ansible/2.8/centos/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM centos:7
+
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
+    && python get-pip.py \
+    && rm get-pip.py
+
+# install latest 2.8 
+RUN pip install "ansible<2.9"
```

Comment:
Could we pin `2.8` instead of asking for lower than `2.9` ?

Reply:
Sure. Done